CHANGELOG.md

```diff
@@ -79,6 +79,14 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
+## Unreleased
```

Comment:
Change this to 2.12.5?

Reply:
I wanted to change this to 2.12.5 when we start releasing 2.12.5. But I think it is also OK to do this in the PR.